### ISSUE TITLE
how to use template in execute-command?

### ISSUE THREAD

**Issue by qwzhou89** [adnanh/webhook#709]
From https://github.com/adnanh/webhook/blob/master/docs/Hook-Examples.md#json-array-payload, I see there is a notation `"execute-command": "{{ .Hookecho }}"`. I want to ask if this `{{ .Hookecho }}` notation is valid? Where do I specify this `Hookecho`?

**Comment by qwzhou89** [comment]
https://github.com/adnanh/webhook/blob/f89b09bef6d8d4d9d27728265fea1166aecc0fbd/internal/hook/hook.go#L769
```go
	if asTemplate {
		funcMap := template.FuncMap{"getenv": getenv}

		tmpl, err := template.New("hooks").Funcs(funcMap).Parse(string(file))
		if err != nil {
			return err
		}

		var buf bytes.Buffer

		err = tmpl.Execute(&buf, nil)
		if err != nil {
			return err
		}

		file = buf.Bytes()
	}
``` 
Because of `tmpl.Execute(&buf, nil)`, it seems that `{{ .Hookecho }}` cannot be used, right?

**Comment by moorereason** [comment]
Correct. Hookecho is a template function used in the unit tests. It's not available in the production binary.

**Comment by qwzhou89** [comment]
> Correct. Hookecho is a template function used in the unit tests. It's not available in the production binary.

Thank you for your reply, I understand now.
